src/models/CommonModel.ts

```diff
@@ -128,6 +153,7 @@ export class CommonModel extends CommonSchema<CommonModel> {
   getFromSchema<K extends keyof Schema>(key: K) {
     let schema = this.originalSchema || {};
     if (typeof schema === 'boolean') schema = {};
+    // eslint-disable-next-line security/detect-object-injection
     return schema[key];
```

Comment:
Can't do `$ {key}`? Do you have some errors here?

Reply:
I am getting `Element implicitly has an 'any' type because expression of type 'string' can't be used to index type Schema'. No index signature with a parameter of type 'string' was found on type 'Schema'.ts(7053)`

Reply:
ahhh, yes, inferring types from object.